### PR TITLE
updating SI Units

### LINKED ISSUE #25: Update docs for pressure reading in hectopascals
This library returns the pressure reading in hectopascals per the CP Design Guide:
https://docs.circuitpython.org/en/latest/docs/design_guide.html#sensor-properties-and-units

Code is here for ref:
https://github.com/adafruit/Adafruit_CircuitPython_MPL3115A2/blob/b32af7d72e016762632f656a891e847058ae41ae/adafruit_mpl3115a2.py#L218-L219

Some of the docs are still saying Pascals.

### PR DESCRIPTION
**Pull request by jposada202020**
closes #25

### CODE REVIEW

File: adafruit_mpl3115a2.py [hunk 1]
```diff
@@ -278,13 +278,13 @@ def temperature(self):
...
         # Read the sea level pressure in bars.
         self._read_into(_MPL3115A2_BAR_IN_MSB, self._BUFFER, count=2)
-        # Reconstruct 16-bit value and scale back to pascals.
+        # Reconstruct 16-bit value and scale back to Hectopascals.
         pressure = (self._BUFFER[0] << 8) | self._BUFFER[1]
-        return pressure * 2.0
+        return pressure * 2.0 / 100
 
     @sealevel_pressure.setter
     def sealevel_pressure(self, val):
```
Comment (tekktrik) [inline adafruit_mpl3115a2.py:290]: The setter for `sealevel_pressure` should also be changed to accept Hectopascals since the getter is being modified.

**Review by tekktrik — CHANGES_REQUESTED** [review]
Looks good, just one small change needed.

File: adafruit_mpl3115a2.py (hunk 1 above)
Comment (jposada202020) [reply adafruit_mpl3115a2.py:290]: please refer to https://github.com/adafruit/Adafruit_CircuitPython_MPL3115A2/pull/27/commits/a416817688da335c8b5f1aa45a50990e087094d0.

Thanks :)

**Review by tekktrik — APPROVED** [review]
Awesome, thank you!

**Pull request merged**